### ISSUE TITLE
Не выходит сообщение об ошибке при попытке загрузить большую картинку для аватарки

### ISSUE THREAD

**Issue by Svetlana678** [Svetlana678/testotvetmail#1]
Как воспроизвести:
1. Открыть главную страницу проекта otvet.mail.ru
2. Перейти в Личный кабинет.
3. В левом меню выбрать пункт "Изменить аватарку".
4. Выбрать фото для загрузки с компьютера размером 9,7 МВ
Фактический результат:
Фото не загружается, бесконечно идет загрузка, сообщение об ошибке не выходит.
Ожидаемый результат:
Появляется сообщение об ошибке при попытке загрузить слишком большой файл.
![аватар ответ](https://user-images.githubusercontent.com/69030252/89062888-e67cf180-d36f-11ea-9309-cd71e8f09bd7.JPG)

**Comment by kostyagina** [comment]
Да, ошибка. Порекомендовать им прописать ограничение по объему файла, либо исправить этот дефект - ошибка функционала.
